xbmc/cores/VideoPlayer/DVDFileInfo.cpp

```diff
@@ -85,11 +87,11 @@ int DegreeToOrientation(int degrees)
   }
 }
 
-bool CDVDFileInfo::ExtractThumb(const CFileItem& fileItem,
-                                CTextureDetails &details,
-                                CStreamDetails *pStreamDetails,
-                                int64_t pos)
+std::unique_ptr<CTexture> CDVDFileInfo::ExtractThumbToTexture(const CFileItem& fileItem)
```

Comment:
This is practically a duplicate of `ExtractThumb` that does not deal with stream details and returns a texture instead of saving directly to file. The old method is still used by chapter thumbnail extraction, until that is fixed up to the new style.